spanner/opentelemetry/src/main/java/com/example/spanner/OpenTelemetryUsage.java

```diff
@@ -0,0 +1,140 @@
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+
+/**
+ * This sample demonstrates how to configure OpenTelemetry and inject via Spanner Options.
+ */
+public class OpenTelemetryUsage {
+
+  public static void main(String[] args) {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project";
+    String instanceId = "my-instance";
+    String databaseId = "my-database";
+
+    // [START spanner_opentelemetry_usage]
+    // Enable OpenTelemetry metrics and traces before Injecting OpenTelemetry
+    SpannerOptions.enableOpenTelemetryMetrics();
+    SpannerOptions.enableOpenTelemetryTraces();
+
+    // Create a new meter provider
+    SdkMeterProvider sdkMeterProvider = SdkMeterProvider.builder()
+        // Use Otlp exporter or any other exporter of your choice.
+        .registerMetricReader(
+            PeriodicMetricReader.builder(OtlpGrpcMetricExporter.builder().build()).build())
+        .build();
+
+    // Create a new tracer provider
+    SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
+        // Use Otlp exporter or any other exporter of your choice.
+        .addSpanProcessor(SimpleSpanProcessor.builder(OtlpGrpcSpanExporter
+            .builder().build()).build())
+            .build();
+
+    // Configure OpenTelemetry object using Meter Provider and Tracer Provider
+    OpenTelemetry openTelemetry = OpenTelemetrySdk.builder()
+        .setMeterProvider(sdkMeterProvider)
+        .setTracerProvider(sdkTracerProvider)
+        .build();
+
+    // Inject OpenTelemetry object via Spanner options or register as GlobalOpenTelemetry.
+    SpannerOptions options = SpannerOptions.newBuilder()
+        .setOpenTelemetry(openTelemetry)
+        .build();
+    Spanner spanner = options.getService();
+
+    // [END spanner_opentelemetry_usage]
+    DatabaseClient dbClient = spanner
+        .getDatabaseClient(DatabaseId.of(projectId, instanceId, databaseId));
+
+    captureQueryStatsMetric(openTelemetry, dbClient);
+    captureGfeMetric(dbClient);
+    sdkMeterProvider.forceFlush();
+    sdkTracerProvider.forceFlush();
+  }
+
+
+  // [START spanner_opentelemetry_capture_query_stats_metric]
+  static void captureQueryStatsMetric(OpenTelemetry openTelemetry, DatabaseClient dbClient) {
+    // Register query stats metric.
+    Meter meter = openTelemetry.getMeter("cloud.google.com/java");
+    DoubleHistogram queryStatsMetricLatencies =
+        meter
+            .histogramBuilder("spanner/query_stats_elapsed")
+            .setDescription("The execution of the query")
+            .setUnit("ms")
+            .build();
```

Comment:
I guess the registration only needs to be done once. If that is the case, can you add a comment stating the same?